lib/stormpath-sdk/resource/login_attempt.rb

```diff
@@ -0,0 +1,5 @@
+class Stormpath::Resource::LoginAttempt < Stormpath::Resource::Instance
```

Comment:
We need this class. It’s a little complicated.
when you define 
```ruby
has_many :login_attempts, can: :create
```
has_many is a method call in association.rb. has_many is a alias for resource_prop_reader method
there in the first two lines you can see
https://github.com/stormpath/stormpath-sdk-ruby/blob/master/lib/stormpath-sdk/resource/associations.rb#L25
that the class based on name is being instantiated

so if we have
```ruby
has_many :login_attempts
```
we need to have a class LoginAttempt which is going to store the response